sys/arm64/arm64/locore.S

```diff
@@ -223,58 +379,43 @@ virtdone:
  * The data in x0 is passed straight to init_secondary.
  */
 ENTRY(mpentry)
+#ifdef __CHERI_PURE_CAPABILITY__
+	.arch_extension noc64
+	.arch_extension a64c
+#endif
 	/* Disable interrupts */
 	msr	daifset, #2
 
 	/* Drop to EL1 */
 	bl	drop_to_el1
```

Comment:
As with _start

Reply:
Don't we need to repeat it here because the `morello_init` macro will switch mode?

Reply:
I meant that this is an A64 bl

Reply:
Ah yes, indeed.